src/Categories/Double/Core.agda

```diff
@@ -261,19 +290,8 @@ interchange law:
       (s₂ : Sq₂ hT₂ hM₂ vM₁ vR₁)
       (s₁ : Sq₂ hT₁ hM₁ vL₁ vM₁) →
       Sq≈ ((s₄ ∘₂ₕ s₃) ∘₂ᵥ (s₂ ∘₂ₕ s₁)) ((s₄ ∘₂ᵥ s₂) ∘₂ₕ (s₃ ∘₂ᵥ s₁))
-  -- When a category is quantified, it is convenient to refer to the levels from a module,
-  -- so we do not have to explicitly quantify over a category when universe levels do not
-  -- play a big part in a proof (which is the case probably all the time).
-  o-level : Level
-  o-level = o
-
-  ℓ-level : Level
-  ℓ-level = ℓ
-
-  e-level : Level
-  e-level = e
 
-  dual : Category o ℓ e
+  dual : Category o ℓ ℓ' e e'
```

Comment:
I'm surprised that this `dual` didn't require the levels be permuted?

Reply:
Quite right and good spot, the levels were all messed up in the definition.